src/main/views/template.njk

```diff
@@ -9,6 +9,32 @@
 
 {% block header %}
 
+{% if isAuthenticated %}
```

Comment:
Nice, I was just looking on 877 how to pass user context, I guess I can re-use your solution